### PR TITLE
:sparkles: Implement logger

### LINKED ISSUE #11: Add a log system
[Pino](https://github.com/pinojs/pino) looks cool

### PR DESCRIPTION
**Pull request by UltiRequiem**
Will write the logs to a `logs.txt` file 🚀 

Will close #11

### CODE REVIEW

File: src/lib/logger.ts [hunk 1]
```diff
@@ -1,7 +1,3 @@
-import { configure, getLogger } from 'log4js';
+import pino from 'pino';
 
-configure('data.log');
-
-const logger = getLogger('logger');
-
-export default logger;
+export default pino(pino.destination('./logs.txt'));
```
Comment (Brian3647) [inline src/lib/logger.ts:3]: Shouldn't it be logs.log?

File: .gitignore [hunk 2]
```diff
@@ -6,3 +6,4 @@ node_modules
 .env
 .env.*
 !.env.example
+logs.txt
```
Comment (Brian3647) [inline .gitignore:9]: Same here

**Review by Brian3647 — CHANGES_REQUESTED** [review]

**Comment by Brian3647** [comment]
Also I'd would write to logs/latest.log and create a back-up of logs

**Comment by UltiRequiem** [comment]
How would you create a backup of the logs?

About the name maybe I could save the file by its date + min started the app so they don't override themselves?

eg -> `logs/09-02-2022-15-59.log`

File: src/lib/logger.ts (hunk 1 above)
Comment (UltiRequiem) [reply src/lib/logger.ts:3]: makes sense

**Comment by Brian3647** [comment]
Yes, I'd also recommend compress the older logs to avoid extreme disk space occupation

**Comment by Brian3647** [comment]
As an example, [Minecraft servers do this perfectly](https://www.hosthorde.com/clients/index.php?rp=/knowledgebase/44/Understanding-Minecraft-server-log-files.html)

**Comment by Brian3647** [comment]
It isn't necessary though and It's not crucial by now

**Comment by UltiRequiem** [comment]
> As an example, [Minecraft servers do this perfectly](https://www.hosthorde.com/clients/index.php?rp=/knowledgebase/44/Understanding-Minecraft-server-log-files.html)

That's interesting

> It isn't necessary though and It's not crucial by now

Yeah, we could leave that to other PR, you can open an issue of this if you want 👍🏽

**Comment by Brian3647** [comment]
ok, we'll see it

**Review by Brian3647 — APPROVED** [review]

**Pull request merged**